packages/exporter-trace-otlp-http/src/OTLPExporterBase.ts

```diff
@@ -93,6 +98,11 @@ export abstract class OTLPExporterBase<
 
   private _export(items: ExportItem[]): Promise<unknown> {
     return new Promise<void>((resolve, reject) => {
+      setTimeout(() => {
```

Comment:
we need to `unref` this timer so it does not hold the process alive in case of shutdown

Reply:
Done.